### PR TITLE
refactor: remove swiper and use embla carousel instead

### PR DESCRIPTION
**Pull request by jln13x**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-08-15).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

I removed the `swiper` dependency and used [embla-carousel](https://github.com/davidjerleke/embla-carousel) instead which is smaller and cleaner imo (no css imports n such)

### CODE REVIEW

File: www/src/components/landingPage/tweets/tweetSlider.tsx [hunk 1]
```diff
@@ -1,42 +1,77 @@
...
-import { Swiper, SwiperSlide } from "swiper/react";
-
-// Import Swiper styles
-import "swiper/css";
-import "swiper/css/pagination";
-import "swiper/css/autoplay";
-import "../../../styles/swiper.css";
-import { Autoplay, Pagination } from "swiper";
 import type { Tweet } from "./types";
+import useEmblaCarousel from "embla-carousel-react";
```
Comment (juliusmarminge) [inline www/src/components/landingPage/tweets/tweetSlider.tsx:2]: quick look at the docs suggest this doesn't have to be react?

Comment (jln13x) [reply www/src/components/landingPage/tweets/tweetSlider.tsx:2]: ye the `embla-carousel-react` is just a thin wrapper.
we could also implement the whole component in vanilla js if thats preferred?

Comment (juliusmarminge) [reply www/src/components/landingPage/tweets/tweetSlider.tsx:2]: Yea if possible just make it an Astro component

Comment (jln13x) [reply www/src/components/landingPage/tweets/tweetSlider.tsx:2]: We would loose the ability to this component as an island tho, is that what we want?

**Comment by jln13x** [comment]
removed the commit related to adding `prettier-plugin-tailwindcss` again, must have missed #481 last night

**Comment by juliusmarminge** [comment]
Looking good 👍

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**